cli.js is not added to files array in package.json, causes npm install errors

Hi, I'm admittedly a newb to yo, but I just used this generator to create a node package with a cli. It was a great boost to getting started, and getting the module working. But,  after I published it to npm, I tried to add it as a dependency in another project but the install kept failing with this error:

```
$ npm install --save approximate-number
npm ERR! Error: ENOENT, chmod 'c:\Users\IBM_ADMIN\service-dashboard\node_modules\approximate-number\cli.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "c:\\Program Files (x86)\\nodejs\\node.exe" "c:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "approximate-number"
npm ERR! cwd c:\Users\IBM_ADMIN\service-dashboard
npm ERR! node -v v0.10.31
npm ERR! npm -v 1.4.23
npm ERR! path c:\Users\IBM_ADMIN\service-dashboard\node_modules\approximate-number\cli.js
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     c:\Users\IBM_ADMIN\service-dashboard\npm-debug.log
npm ERR! not ok code 0
```

It took me a while, but eventually I figured out that the generator had add a `"files": ["lib/"]` entry in my package.json, and that instructs npm to skip everything else, including the `cli.js` file that the generator put in the project root.

--- a/app/templates/_package.json
+++ b/app/templates/_package.json
@@ -22,7 +22,8 @@
   },
   "license": "<%= props.license %>",
   "files": [
-    "lib"
+    "lib"<% if (props.cli) { %>,
+    "cli.js"<% } %>
   ],
   "keywords": [
     "<%= slugname %>" <%

